### DISCUSSION TITLE
checkPerimeterForResponseCard still needed ?

### DISCUSSION THREAD

**Discussion by freddidierRTE** [opfab/operatorfabric-core#2590] [Ideas]
We have checkPerimeterForResponseCard parameter available to avoid checking perimeter when responding to card .

Everything being control via perimeter , is this option still needed ?

**Comment by didierfred** [comment]
OK

**Comment by geppyz** [comment]
The checkPerimeterForResponseCard parameter can be removed from our point of view.
